.github/workflows/container-build-push-gea.yml

```diff
@@ -75,7 +75,9 @@ on:
         required: false
         default: ""
       prefix:
-        description: "The prefix to use for the image tags."
+        description: |
+          "The prefix to use for the image tags. For example 'some-feature-'. "
```

Comment:
just a tiny detail but would it be clearer all along through the file if the variable name is image_tag_prefix then?

Reply:
the prefix input is already used. i am not sure if `tags-prefix` is a better name because it would not be applied if the tags input is passed. therefore i did choose short input variable names here.